specification/gedcom-01-serialization.md

```diff
@@ -401,12 +401,17 @@ defines the following tags
 | :---- | :---- |
 | `_SKYPEID` | `http://xmlns.com/foaf/0.1/skypeID` |
 | `_MEMBER` | `http://xmlns.com/foaf/0.1/member` |
+
+Note that at the time of writing, the [FOAF](https://xmlns.com/foaf/spec/20140114.html) URIs used in this example are not URLs.
```

Comment:
This text might not be the best.

- We could expand on it, explaining that URI meanings survive changes such as the reconfiguring of the xmlns.com host in 2022 that made these URIs no longer redirect to the FOAF spec.
- We could replace the FOAF examples with another semantic web ontoloy like schema.org that hasn't (yet) had [link rot](https://en.wikipedia.org/wiki/Link_rot).
- We could use `example.com` URIs instead. That might be preferable because while FOAF does define skypeID, it does so without reference to GEDCOM and hence without indicating whether it's a structure type or enumerated value, where it can appear, etc.
- We could wait to update this until a 7.0 extension is published so we can use a real GEDCOM example

I don't have strong feelings on which (if any) of the above options is better than the PR's current text